tasks/configure-Debian.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Create /etc/default/kibana
+  template:
+    src: Ubuntu.default.j2
+    dest: /etc/default/kibana
+    # any files under /etc/default should be mode 644, but the package installs
+    # the file as mode 0664
```

Comment:
is it a bug or is there any reason that the package sets 0664?
bug number should be written if it has been reported as a bug.